### PR TITLE
Add team entry for Jacob Zhi

### PR DESCRIPTION
**Pull request by jaczhi**

### CODE REVIEW

**Comment by reginawang3495** [comment]
sorry for the extremely late review -- we actually need images to be 512x512 px :0

File: _team/jzhi.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Jacob Zhi"
+group: "member"
+title: "React Team"
```
Comment (reginawang3495) [inline _team/jzhi.md:4]: I'm not sure you'd want to make a PR every time you change?
```suggestion
title: "Developer"
```

**Review by reginawang3495 — APPROVED** [review]
lgtm! are you sure you don't want to add a pic?

**Comment by jaczhi** [comment]
Can't find a good pic as of yet 😅, merging with the change.

**Pull request merged**